cmake/therock_amdgpu_targets.cmake

```diff
@@ -54,7 +54,7 @@ therock_add_amdgpu_target(gfx90a "MI210/250 CDNA" FAMILY gfx90X-all dcgpu-all gf
 therock_add_amdgpu_target(gfx942 "MI300A/MI300X CDNA" FAMILY dcgpu-all gfx94X-all gfx94X-dcgpu)
 
 # gfx950
-therock_add_amdgpu_target(gfx950 "" FAMILY dcgpu-all gfx950-all gfx950-dcgpu)
+therock_add_amdgpu_target(gfx950 "MI350X/MI355X CDNA" FAMILY dcgpu-all gfx950-all gfx950-dcgpu)
```

Comment:
Ah, I noticed the same thing and sent https://github.com/ROCm/TheRock/pull/1387 yesterday. Hope you don't mind me closing this and merging that one instead